docs/getting-started/install/linux/index.md

```diff
@@ -12,7 +12,7 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 Debian, Ubuntu, and derivatives
```

Comment:
I'd like something similar to https://www.microsoft.com/net/core#linuxredhat where you choose your distro first so the page is smaller.

Should be easily doable given we already use such a switcher on https://github.com/mono/website/blob/gh-pages/download/index.html#L23, just horizontal.